api/src/wrangler.rs

```diff
@@ -29,6 +25,7 @@ pub struct ExecutionWrangler {
     gas_limit: u64,
     gas_fee_cap: TokenAmount,
     gas_premium: TokenAmount,
+    circulating_supply: RefCell<TokenAmount>,
```

Comment:
This is an odd place for this field to be. I would expect it to be at least down in the underlying bench, if not deeper. Setting this value has to affect WASM-compiled actor code that executes on the workbench when it calls the syscall to fetch this value. So putting it here is probably wrong.

Reply:
As discussed, this should go along with fields like epoch, base fee. In the FVM implementation these fields are in the MachineContext or NetworkConfig held by the machine.

Reply:
I've deferred exposing NetworkConfig for now as it is a concept owned by the short-lived Kernel during execution and likely not relevant for exposing.

MachineContext has a large amount of overlap. Like the rename predictable_address -> delegated_address. I'll make the change in vm_api (builtins repo) first and have the changes flow here in a followup.